astropy/stats/sigma_clipping.py

```diff
@@ -144,21 +150,14 @@ class SigmaClip:
 
     cenfunc : {'median', 'mean'} or callable, optional
         The statistic or callable function/object used to compute the
-        center value for the clipping.  If set to ``'median'`` or
-        ``'mean'`` then having the optional `bottleneck`_ package
-        installed will result in the best performance.  If using a
-        callable function/object and the ``axis`` keyword is used, then
-        it must be callable that can ignore NaNs (e.g., `numpy.nanmean`)
-        and has an ``axis`` keyword to return an array with axis
-        dimension(s) removed.  The default is ``'median'``.
-
-        .. _bottleneck:  https://github.com/pydata/bottleneck
+        center value for the clipping. If using a callable function/object and
+        the ``axis`` keyword is used, then it must be callable that can ignore
```

Comment:
"then it must be callable that can ignore" -> "then it must be able to ignore"

Reply:
Fixed